Doc/tools/.nitignore

```diff
@@ -36,7 +36,6 @@ Doc/library/smtplib.rst
 Doc/library/socket.rst
 Doc/library/ssl.rst
 Doc/library/stdtypes.rst
-Doc/library/subprocess.rst
 Doc/library/termios.rst
```

Comment:
A few more warnings in this branch:
```
Doc/library/subprocess.rst
  1446: py:func reference target not found: os.popen2 [ref.func]
  1446: py:func reference target not found: os.popen3 [ref.func]
  1516: py:class reference target not found: popen2.Popen3 [ref.class]
  1516: py:class reference target not found: popen2.Popen4 [ref.class]
  1658: py:const reference target not found: subprocess._USE_VFORK [ref.const]
  1666: py:const reference target not found: subprocess._USE_POSIX_SPAWN [ref.const]
```

So let's just keep the ignore in this old one:

```suggestion
Doc/library/subprocess.rst
Doc/library/termios.rst
```